cluster/addons/cluster-monitoring/googleinfluxdb/heapster-controller-combined.yaml

```diff
@@ -4,8 +4,8 @@
   {% set heapster_memory = (200 + num_nodes * 3)|string + "Mi" -%}
 {% endif -%}
 
-apiVersion: v1
-kind: ReplicationController
+apiVersion: extensions/v1beta1
+kind: Deployment
 metadata:
   name: heapster-v1.0.0-beta1
```

Comment:
Deployment's should not have version string in their name. Deployments objects last longer than a single version of a replicaset.